components/table/demo/table-test.js

```diff
@@ -102,6 +102,12 @@ class TestTable extends RtlMixin(DemoPassthroughMixin(TableWrapper, 'd2l-table-w
 						<tr>
 							<th scope="col" sticky><d2l-selection-select-all></d2l-selection-select-all></th>
 							${this._renderDoubleSortButton('Location')}
+							<th scope="col" colspan="${columns.length}" sticky>
+								Metrics
+							</th>
+						</tr>
+						<tr>
+							<th scope="col" sticky></th>
```

Comment:
If we proceed here, I'm wondering if it might make sense to create a specialized test just for this instead of continually bolting onto this existing one (a trend you didn't start!)? I'm just worried that we're now losing our test case for single-row sticky headers.

Reply:
Yeah, that makes sense to me. If we do decide to go this route, I can break this out into two and create some new tests to cover.